### ISSUE TITLE
Notification header text does not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52684]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer

**Repro Steps:**
1. Navigate to Status Bar at the bottom and move to notifications.
2. Move to "Notification" button that appears in the status bar.
3. Navigate there when there are no new notifications/ also when there are notifications with source defined.

**Actual:**
Color contrast ratio is less than 4.5:1 for the "No new notification" content and "C# Extension" present inside the notification window present on the screen.

**Expected:**
Color contrast ratio should be more than 4.5:1 for the "No new notification" content and "C# Extension" present inside the notification window present on the screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The low vision/visually impaired users will not be able to view the content properly.

**MAS Reference:**
MAS 1.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment For Reference:**

![a11y_vscode_statusbar_colorcontrast1](https://user-images.githubusercontent.com/39204909/41777918-99b81520-764a-11e8-9e29-69049a96c748.png)

![a11y_vscode_statusbar_colorcontrast2](https://user-images.githubusercontent.com/39204909/41777919-99e7f39e-764a-11e8-8cf2-169f043d617a.png)

Does this issue occur when all extensions are disabled?: Yes
